force-app/main/triggers/classes/HOT_UserHandler.cls

```diff
@@ -14,4 +15,28 @@ public without sharing class HOT_UserHandler extends MyTriggers {
             }
         }
     }
-}
+
+    public override void onAfterUpdate(Map<Id, sObject> triggerOldMap) {
+        Map<Id, User> oldUserMap = (Map<Id, User>)Trigger.oldMap;
+        Map<Id, User> newUserMap = (Map<Id, User>)Trigger.newMap;
+    
+        List<Profile> formidlerProfile = [SELECT Id, Name FROM Profile WHERE Name = 'HOT Tolk Formidler'];
+        List<Profile> ansattProfile = [SELECT Id, Name FROM Profile WHERE Name = 'HOT Tolk Ansatt'];
+    
+        Set<Id> usersWithChangedProfile = new Set<Id>();
+    
+        for (Id userId : oldUserMap.keySet()) {
+            User oldUser = oldUserMap.get(userId);
+            User newUser = newUserMap.get(userId);
+    
+            if (oldUser.ProfileId == formidlerProfile[0].Id && newUser.ProfileId == ansattProfile[0].Id) {
```

Comment:
Sidene profilene ikke finnes når man lager pakka (bare ting i main blir med), så feiler dette. Du bør legge til en sjekk på om den finner profilene, og hvis ikke bare hoppe over logikken.

Du kan også endre logikken til å sjekke om profilene endrer deg, og at den endrer seg til Tolk. Er ikke så farlig hva den endrer seg fra, og vi bør støtte flere en Formidler.

Reply:
Har lagt til støtte for alle profiler: https://github.com/navikt/crm-hot/pull/1876/commits/bde53376999348e33001f2077a73079908e71415

Og en sjekk for å se om profilene finnes: https://github.com/navikt/crm-hot/pull/1876/commits/776e126ca4fe332b48443b206f34bacef3982ad6